internal/driver/driver.go

```diff
@@ -233,7 +282,13 @@ func (d *Driver) HandleReadCommands(deviceName string, protocols map[string]mode
 			}
 			cv, err = sdkModels.NewCommandValue(req.DeviceResourceName, common.ValueTypeObject, data)
 		case VideoStreamUri:
-			cv, err = sdkModels.NewCommandValue(req.DeviceResourceName, req.Type, device.rtspUri)
+			rtspUri := &url.URL{
```

Comment:
i think we should revert this back to how it is, and when we go to start streaming do the user/pass lookup there instead.